src/models/MetaModel.ts

```diff
@@ -2,8 +2,13 @@ export interface MetaModelOptionsConst {
   originalInput: unknown;
 }
 
+export interface MetaModelOptionsDiscriminator {
+  originalInput: string;
```

Comment:
What does the `originalInput` refer to here? 🤔 Just the value from the input or?

Reply:
Yeah, just the value from the input. Perhaps there is a better name for it. Just call it discriminator or value?

Reply:
I think that is better yea 🤔

Reply:
Changed it to `discriminator`